simde/x86/avx512/conflict.h

```diff
@@ -0,0 +1,39 @@
+#if !defined(SIMDE_X86_AVX512_CONFLICT_H)
+#define SIMDE_X86_AVX512_CONFLICT_H
+
+#include "types.h"
+#include "mov_mask.h"
+
+HEDLEY_DIAGNOSTIC_PUSH
+SIMDE_DISABLE_UNWANTED_DIAGNOSTICS
+SIMDE_BEGIN_DECLS_
+
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m128i
+simde_mm_conflict_epi32 (simde__m128i a) {
+  #if defined(SIMDE_X86_AVX512VL_NATIVE) && defined(SIMDE_X86_AVX512CD_NATIVE)
+    return _mm_conflict_epi32(a);
+  #else
+    simde__m128i_private
+      r_ = simde__m128i_to_private(simde_mm_setzero_si128()),
+      a_ = simde__m128i_to_private(a);
+
+    #if !defined(__INTEL_COMPILER)
+      SIMDE_VECTORIZE
+    #endif
+    for (size_t i = 1 ; i < (sizeof(r_.i32) / sizeof(r_.i32[0])) ; i++) {
+      r_.i32[i] = HEDLEY_STATIC_CAST(int32_t, simde_mm_movepi32_mask(simde_mm_cmpeq_epi32(simde_mm_set1_epi32(a_.i32[i]), a))) & ((1 << i) - 1);
```

Comment:
Be careful with functions like `simde_mm_movepi32_mask` which return or use a predicate type (`__mmask*`); they are usually quite slow outside of AVX-512.

Luckily this particular function can be emulated pretty well using `_mm_movemask_ps` on x86, and AArch64 isn't *too* bad, so in this case it's okay.  I've just noticed you doing similar things for other functions, too, and there are some pretty big performance cliffs here so I wanted to warn you :)